src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -431,6 +432,14 @@ public Index GetIndex(string name)
 
                 return instance;
             }
+            catch (TimeoutException toe)
+            {
+                throw new IndexCreationException($"Failed to create auto index: {definition.Name}, the cluster is probably down.", toe);
```

Comment:
I think we should do the same thing for static indexes?

Reply:
Yes.

Reply:
A static index is something that you deploy and we don't allow to deploy new indexes while the cluster is down since it is a raft command. 
Querying is something we allow when the cluster is down, since this error can come as a surprise for the user who thinks he is not doing a cluster operation, we need to be more clear.

Reply:
It still deserves a better error.

Reply:
In general, there is no situation in which I'm every going to say, we don't need good error handling here.

Reply:
Also for consistency it would be better to throw same kind of exception both for static and auto.